pennylane/workflow/execution.py

```diff
@@ -135,41 +131,41 @@ def _get_ml_boundary_execute(
         pennylane.QuantumFunctionError if the required package is not installed.
 
     """
-    mapped_interface = INTERFACE_MAP[interface]

```

Comment:
At this point we're assuming that the `interface` is already mapped to one of the canonical interface names.

Reply:
👍